pkg/deployments/tasks/2022xxxx-gauge-controller/index.ts

```diff
@@ -0,0 +1,16 @@
+import Task from '../../src/task';
+import { TaskRunOptions } from '../../src/types';
+import { GaugeSystemDeployment } from './input';
+
+export default async (task: Task, { force, from }: TaskRunOptions = {}): Promise<void> => {
+  const input = task.input() as GaugeSystemDeployment;
+
+  const veBPTArgs = [input.BPT, 'Vote Escrowed Balancer BPT', 'vebptBAL', '1'];
```

Comment:
The version argument will be removed by https://github.com/balancer-labs/balancer-v2-monorepo/pull/1103